lib/ui/src/modules/ui/components/down_panel/index.js

```diff
@@ -4,9 +4,9 @@ import style from './style';
 
 class DownPanel extends Component {
   renderTab(name, panel) {
-    let tabStyle = style.tablink;
+    let tabStyle = style(this.props.theme).tablink;
```

Comment:
maybe you'll define the style once, instead of creating it in each call ? 
something like 
```javascript
import commonStyle from './style';
/**/
const style = commonStyle(this.props.theme); 
// in this case you will have less conflicts and irrelevant changes 
/**/
```